extra_data/components.py

```diff
@@ -1461,9 +1464,19 @@ class JUNGFRAU(MultimodDetectorBase):
     module_shape = (512, 1024)
 
     def __init__(self, data: DataCollection, detector_name=None, modules=None,
-                 *, min_modules=1, n_modules=None):
+                 *, min_modules=1, n_modules=None, st_modno=1):
         super().__init__(data, detector_name, modules, min_modules=min_modules)
 
+        if st_modno:
```

Comment:
This branch will always run, right? Unless someone makes a `JNGFR00`, but in that case it should run, because we want JUNGFRAU numbering to start from 1 for EXtra-geom.

So I think we can lose the `if`.

Reply:
Good point, this condition is removed. [37cfa0874adac26d2df26f8a4c429ab745111aaa](https://github.com/European-XFEL/EXtra-data/pull/379/commits/37cfa0874adac26d2df26f8a4c429ab745111aaa)